tests/zfs-tests/include/blkdev.shlib

```diff
@@ -462,13 +462,15 @@ function unload_scsi_debug
 # Get scsi_debug device name.
 # Returns basename of scsi_debug device (for example "sdb").
 #
-function get_debug_device
+function get_debug_device #num
 {
```

Comment:
Just to add a little detail, could you have the comment say:
```
# Get scsi_debug device name.
# Returns basename of scsi_debug device (for example "sdb").
#
# $1 (optional): Return the first $1 number of SCSI debug device names.
function get_debug_device #num
{
```

Reply:
Thanks, taken verbatim.